File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfActionPermission.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdAction;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioMapOfActionPermission extends StudioBase
{
  public MetaIdAction[] keys;

  public Map<MetaIdAction, StudioDtoActionPermission> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfActionPermission.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdAction

interface StudioMapOfActionPermission : StudioBase {
    val keys: List<MetaIdAction>
    val map: Map<MetaIdAction, StudioDtoActionPermission>
}
